engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -85,6 +85,31 @@ <h4 translate>Subscriptions</h4>
             on-save="saveSubscriptionFacet(host)">
         </dd>
 
+        <dt translate>Role</dt>
```

Comment:
This is all functional but the next build of candlepin we get (hopefully tomorrow) will have an API to get the syspurpose values coming through the manifest for the org so we'll give those options for role, usage, addons. In the future we might make these freetext or something in support of Custom Purpose Attributes

Reply:
I had imagined it being a freetext field with autocomplete from Candlepin.  Selecting it from the values in Candlepin would be easier, I suppose, but what if a user sets a value to "ASDF" from purposectl, would Candlepin then expose that to us or only values from the manifest?

Reply:
Good point. I had asked Candlepin to create an API that would also give us the purpose values set across consumers in an org, but for reasons we won't have that in the first go-round. I'm OK with that cause this is primarily in support of RHEL8 so as long as we can get the manifest values (that's what the API they built does) then we're good. Yeah - we'll combine API response plus whatever they've set via purposectl/whatever.

I'm OK with the autocomplete approach as long as it doesn't leave too much guesswork - kind of like how scoped search shows you the searchable fields when you click in the text field if that's what you were thinking too.